patches/jsc.patch

```diff
@@ -1,3 +1,14 @@
+diff -aur target-org/webkit/Source/bmalloc/bmalloc/DebugHeap.cpp target/webkit/Source/bmalloc/bmalloc/DebugHeap.cpp
+--- target-org/webkit/Source/bmalloc/bmalloc/DebugHeap.cpp      2017-08-01 10:21:09.104840436 +0200
++++ target/webkit/Source/bmalloc/bmalloc/DebugHeap.cpp  2017-07-31 17:18:01.935186338 +0200
+@@ -26,6 +26,7 @@
+ #include "DebugHeap.h"
+ #include "BAssert.h"
+ #include "BPlatform.h"
++#include <posix_memalign.h>
```

Comment:
If we are patching this I'd actually prefer if we just remove the place where posix_memalign is being called and add an assert(false) there to make sure the app would crash when it gets to that place (we claim that this code is not being used as it is only used when JSC debugging is enabled)